components/templates/common/Base.tsx

```diff
@@ -2,9 +2,11 @@ import React from 'react'
 import { Header } from '@/components/organisms/Header'
 import { Footer } from '@/components/organisms/Footer'
 import { BottomRightCircleStyle, InnerStyle, OuterStyle, UpLeftCircleStyle } from './style'
+import { Container } from '@/components/atoms/Container'
 
 type PropsType = {
   children: React.ReactNode
+  // eslint-disable-next-line no-unused-vars
   onClickMenuItem: (item: string) => void
```

Comment:
This is necessary because the goal is to ignore warnings that are already noticed and are not important.
If there is a way to avoid this warning when passing arguments to a function of type void, I would adopt it.

Reply:
Seems it not works as expected.
```
9:21  error  '_item' is defined but never used  no-unused-vars
```

Reply:
Thanks.
Updated the ESLint configuration as follow.
```
{
  "extends": ["eslint:recommended", "next", "next/core-web-vitals", "prettier"],
  "rules": {
    "no-unused-vars": [
      "warn",
      {
        "argsIgnorePattern": "^_",
        "varsIgnorePattern": "^_",
        "caughtErrorsIgnorePattern": "^_",
        "destructuredArrayIgnorePattern": "^_"
      }
    ]
  }
}

```